### ISSUE TITLE
Bug: A table will not move to make space for another.

### ISSUE THREAD

**Issue by batonac** [frappe/print_designer#83]
Ordinarily, print formats made with the print designer allow a table to expand to push following elements onto subsequent pages, as necessary. This is great default functionality, and affords quite a bit of flexibility in accommodating various child table sizes, while retaining a professional design.

In the following screenshot, for example, The items of a Sales Invoice expand to fill most of the second page of a print. Table headers are appropriately repeated on the second page, and the items below the table in the print format are moved to the second page, based on the original spacing from the print format:
![image](https://github.com/frappe/print_designer/assets/4996285/13030604-ad47-4d79-bfa1-e3655e5953e8)

The issue arises when a table element is placed below an expanding table element. In this case, the following table and its siblings refuse to move to subsequent pages, causing a very jumbled printout:
![image](https://github.com/frappe/print_designer/assets/4996285/817f8e6a-a408-41a6-9432-99bb25aeb864)

The case in point is trying to include the Payment Schedule term descriptions in a Sales Invoice Print Format:
![image](https://github.com/frappe/print_designer/assets/4996285/ce4409bf-c37a-4188-8b70-2e914b9bd3b4)

The only way to accomplish this (that I know of) is to include a second table element:
![image](https://github.com/frappe/print_designer/assets/4996285/dc110df1-909b-454e-b940-b81bf33ec01e)

In summary, table fields need to learn to behave better with their own kind 😀

**Comment by maharshivpatel** [comment]
@batonac This is known problem so i have created one issue here https://github.com/frappe/print_designer/issues/102 where you can track the progress and i will also inform you when it is done.

Current Implementation is if there is 1 table put everything below it. so when there are 2 table we do nothing maybe i will add workaround for time being to select any table as main table and that will push everything below it to last page.

I am not sure how long this will take.

Thank you for sharing the Issue.

Closing This issue for now.
